### DISCUSSION TITLE
Debugger is missing 🥲

### DISCUSSION THREAD

**Discussion by MarcellGranat** [posit-dev/positron#5255] [Q&A] [answered]
Hi,

I am quite confused now with the debugger pane, and how I should set that up, but currently it is missing (picture) and I cannot find the solution to that. I believe I should add a custom debugger, but the R debugger extension is banned, with the message "Cannot install the 'R' extension because it conflicts with Positron built-in features.".

How am I supposed to fix this issue?

![Positron 2024-11-04 11 32 02](https://github.com/user-attachments/assets/3c2c4604-6420-4c25-9a3d-b34fa19fa15e)

Thanks 🙏 in advance,

**Comment by DavisVaughan** [comment, marked as answer]
Use `debug()` or `debugonce()` on the function of interest, or insert a `browser()` statement directly into the function of interest.

In the future we will let you set breakpoints directly in the file but that doesn't work yet.

So, like:

```r
library(dplyr)
debugonce(mutate)
mutate(mtcars, x = mpg + cyl)
```
